app/permissions_migrations.go

```diff
@@ -632,5 +728,12 @@ func (a *App) DoPermissionsMigrations() error {
 			return err
 		}
 	}
+	// Need to increase the size of the Permissions column in the Roles table
+	for _, db := range a.Srv().sqlStore.GetAllConns() {
```

Comment:
I'm unsure if this is the right place, do we have an area where we run one time schema update?